frontend/test/__support__/e2e/helpers/e2e-collection-helpers.js

```diff
@@ -20,5 +20,6 @@ export function openNewCollectionItemFlowFor(type) {
 }
 
 export function getSidebarSectionTitle(title) {
-  return cy.findAllByRole("heading", { name: title });
+  const name = new RegExp(title);
+  return cy.findAllByRole("heading", { name });
```

Comment:
Would it be better to specify regex in places where this function is used? I don't think it'd obvious for everyone that title is a regex otherwise:

```
getSidebarSectionTitle(/Bookmarks/);
``` 

If that would be ok, no change here is needed.

Reply:
@ranquild Yes, cleaner, thank you.